### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by mahdimahdawi**
**This PR Contain: **

new directory for modules files added to the project.
index.js is the entry point of the program for js file.
All modules imported inside index.js file. 
index.html and index.css for entry point to project.
package.json file added to the project with the help of npm init -y.
Method and functions changed to arrow function.
Imported luxon package as a module and used to show the local time.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
+/* eslint-disable */
```
Comment (HAMMAS-SALEEM) [inline index.js]: - [x] Please enable linters in your index.js file :blush: I know there are some errors. You can fix these errors by quick fix I have given below :arrow_down:
quick-fix

File: modules/bookClass.js [hunk 2]
```diff
@@ -36,6 +35,7 @@ export class BookClass {
 }
```
Comment (HAMMAS-SALEEM) [inline modules/bookClass.js:35]: - [x] While adding or removing a book your page reloads. You have made a Single Page Application that is not expected to reload. Please fix this issue. :+1:

File: modules/ui.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ const localTime = document.querySelector('.date-time');
 const listPage = document.querySelector('.book-list-page');
 const AddPage = document.querySelector('.add-book');
 const contactPage = document.querySelector('.contact');
-const navBar = document.querySelectorAll('.item-link');
```
Comment (HAMMAS-SALEEM) [inline modules/ui.js:10]: - [x] When your website loads then the default page is Contact Page. Please make List your default page :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @mahdimahdawi :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Following Linters
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions
- [x] Please make your commit messages more meaningful to make your pull request in a [professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) :blush:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @mahdimahdawi               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**